source/ingress.go

```diff
@@ -152,7 +160,11 @@ func endpointsFromIngress(ing *v1beta1.Ingress) []*endpoint.Endpoint {
 			continue
 		}
 
-		endpoints = addEndpointsFromTargetAnnotation(ing, rule.Host, endpoints)
+		endpoints = getEndpointsFromTargetAnnotation(ing, rule.Host)
```

Comment:
I am not entirely sure about this, so this basically ignores all but last`rules.host` and generate endpoint from it. Why not generate an endpoint for each of the hosts found ?